index.bs

```diff
@@ -2906,14 +2897,17 @@ partial dictionary MLOpSupportLimits {
         1. If |options|.{{MLConvTranspose2dOptions/bias}} [=map/exists=]:
             1. If its [=MLOperand/shape=] is not equal to « |outputChannels| », then [=exception/throw=] a {{TypeError}}.
             1. If its [=MLOperand/dataType=] is not one of its [=/allowed data types=] (according to [this table](#constraints-convTranspose2d)), then [=exception/throw=] a {{TypeError}}.
-        1. If |options|.{{MLConvTranspose2dOptions/outputSizes}} [=map/exists=], let |outputSizes| be |options|.{{MLConvTranspose2dOptions/outputSizes}}.
-        1. Otherwise, let |outputSizes| be the result of [=MLGraphBuilder/calculating convtranspose2d output sizes=] given |inputHeight|, |inputWidth|, |filterHeight|, |filterWidth|, |options|.{{MLConvTranspose2dOptions/padding}}, |options|.{{MLConvTranspose2dOptions/strides}}, |options|.{{MLConvTranspose2dOptions/dilations}}, and |options|.{{MLConvTranspose2dOptions/outputPadding}}.
+        1. If |options|.{{MLConvTranspose2dOptions/outputSizes}} [=map/exists=], then:
+            1. Let « |outputHeight|, |outputWidth| » be |options|.{{MLConvTranspose2dOptions/outputSizes}}.
```

Comment:
Old issue, the `|options|.{{MLConvTranspose2dOptions/outputSizes}}` need to be validated, Chromium prototype does: https://source.chromium.org/chromium/chromium/src/+/main:services/webnn/public/cpp/graph_validation_utils.cc;l=755;drc=8636bb104c0e28b695adfe050dd1b70bfe05cdc8;bpv=1;bpt=1

I'll open an issue if you agree.

Reply:
Issue is good. I have a local branch with a bunch of those validation fixes, was waiting until this merged to rebase/upload a PR

Reply:
Opened issue https://github.com/webmachinelearning/webnn/issues/818